micro/cli update broke users and storage-root service

```
 .\ocis-reva.exe users
{"level":"info","service":"reva","time":"2020-02-04T11:19:34+01:00","message":"Continue without config"}
panic: interface conversion: flag.Value is *flag.stringValue, not *cli.StringSlice

goroutine 1 [running]:
github.com/micro/cli/v2.lookupStringSlice(...)
        C:/Users/jfd/go/pkg/mod/github.com/micro/cli/v2@v2.1.1/flag_string_slice.go:152
github.com/micro/cli/v2.(*Context).StringSlice(0xc000487080, 0x11b383b, 0x7, 0xc0001b8ab0, 0x4, 0x0)
        C:/Users/jfd/go/pkg/mod/github.com/micro/cli/v2@v2.1.1/flag_string_slice.go:144 +0x10b
github.com/owncloud/ocis-reva/pkg/command.Users.func1(0xc000487080, 0x1d, 0x38)
        C:/Users/jfd/test/ocis-reva/pkg/command/users.go:26 +0x54
github.com/micro/cli/v2.(*Command).Run(0xc0001a8360, 0xc000486840, 0x0, 0x0)
        C:/Users/jfd/go/pkg/mod/github.com/micro/cli/v2@v2.1.1/command.go:148 +0x583
github.com/micro/cli/v2.(*App).RunContext(0xc000500300, 0x17ba460, 0xc0000300c0, 0xc000004480, 0x2, 0x2, 0x0, 0x0)
        C:/Users/jfd/go/pkg/mod/github.com/micro/cli/v2@v2.1.1/app.go:302 +0x797
github.com/micro/cli/v2.(*App).Run(...)
        C:/Users/jfd/go/pkg/mod/github.com/micro/cli/v2@v2.1.1/app.go:210
github.com/owncloud/ocis-reva/pkg/command.Execute(0xfa8bc0, 0xc000038118)
        C:/Users/jfd/test/ocis-reva/pkg/command/root.go:102 +0xa19
main.main()
        C:/Users/jfd/test/ocis-reva/cmd/ocis-reva/main.go:10 +0x29
```
and
```
.\ocis-reva.exe storage-root
{"level":"info","service":"reva","time":"2020-02-04T11:10:44+01:00","message":"Continue without config"}
panic: interface conversion: flag.Value is *flag.stringValue, not *cli.StringSlice

goroutine 1 [running]:
github.com/micro/cli/v2.lookupStringSlice(...)
        C:/Users/jfd/go/pkg/mod/github.com/micro/cli/v2@v2.1.1/flag_string_slice.go:152
github.com/micro/cli/v2.(*Context).StringSlice(0xc00003bdc0, 0x11b383b, 0x7, 0xc0001c8090, 0x4, 0x0)
        C:/Users/jfd/go/pkg/mod/github.com/micro/cli/v2@v2.1.1/flag_string_slice.go:144 +0x10b
github.com/owncloud/ocis-reva/pkg/command.StorageRoot.func1(0xc00003bdc0, 0x26, 0x50)
        C:/Users/jfd/test/ocis-reva/pkg/command/storageroot.go:26 +0x54
github.com/micro/cli/v2.(*Command).Run(0xc0000b7c20, 0xc00003b2c0, 0x0, 0x0)
        C:/Users/jfd/go/pkg/mod/github.com/micro/cli/v2@v2.1.1/command.go:148 +0x583
github.com/micro/cli/v2.(*App).RunContext(0xc000074780, 0x17ba460, 0xc0000300b8, 0xc000004480, 0x2, 0x2, 0x0, 0x0)
        C:/Users/jfd/go/pkg/mod/github.com/micro/cli/v2@v2.1.1/app.go:302 +0x797
github.com/micro/cli/v2.(*App).Run(...)
        C:/Users/jfd/go/pkg/mod/github.com/micro/cli/v2@v2.1.1/app.go:210
github.com/owncloud/ocis-reva/pkg/command.Execute(0xfa8bc0, 0xc000038118)
        C:/Users/jfd/test/ocis-reva/pkg/command/root.go:102 +0xa19
main.main()
        C:/Users/jfd/test/ocis-reva/cmd/ocis-reva/main.go:10 +0x29
```

@C0rby may already have an idea what is necessary to fix this.

--- a/changelog/unreleased/76
+++ b/changelog/unreleased/76
@@ -1,0 +1,8 @@
+Bugfix: set correct flag type in the flagsets
+
+While upgrading to the micro/cli version 2 there where two instances of `StringFlag`
+which had not been changed to `StringSliceFlag`.
+This caused `ocis-reva users` and `ocis-reva storage-root` to fail on startup.
+
+https://github.com/owncloud/ocis-reva/issues/75
+https://github.com/owncloud/ocis-reva/pull/76

--- a/pkg/flagset/storageroot.go
+++ b/pkg/flagset/storageroot.go
@@ -116,9 +116,9 @@ func StorageRootWithConfig(cfg *config.Config) []cli.Flag {
 			EnvVars:     []string{"REVA_STORAGE_ROOT_URL"},
 			Destination: &cfg.Reva.StorageRoot.URL,
 		},
-		&cli.StringFlag{
+		&cli.StringSliceFlag{
 			Name:    "service",
-			Value:   "storageprovider",
+			Value:   cli.NewStringSlice("storageprovider"),
 			Usage:   "--service storageprovider [--service otherservice]",
 			EnvVars: []string{"REVA_STORAGE_ROOT_SERVICES"},
 		},

--- a/pkg/flagset/users.go
+++ b/pkg/flagset/users.go
@@ -195,9 +195,9 @@ func UsersWithConfig(cfg *config.Config) []cli.Flag {
 			EnvVars:     []string{"REVA_USERS_URL"},
 			Destination: &cfg.Reva.Users.URL,
 		},
-		&cli.StringFlag{
+		&cli.StringSliceFlag{
 			Name:    "service",
-			Value:   "userprovider", // TODO preferences
+			Value:   cli.NewStringSlice("userprovider"), // TODO preferences
 			Usage:   "--service userprovider [--service otherservice]",
 			EnvVars: []string{"REVA_USERS_SERVICES"},
 		},

